mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -28,6 +28,7 @@ llvm::raw_ostream &mlir::rock::operator<<(llvm::raw_ostream &os,
   case GemmDimension::MorN:
     return os << "GemmDimension::MorN";
   }
+  return os;
```

Comment:
Since you are already passing ostream as reference, it looks redundant to return it.

Reply:
GCC was whining about non-void function potentially not returning a value, thought I'd fix